src/cli.ts

```diff
@@ -5,14 +5,18 @@ import getopts = require('getopts')
 const opts = getopts(process.argv.slice(2), {
   alias: {
     h: 'help',
+    n: ['npmClient', 'npm-client'],
```

Comment:
I don't think we need to aliases. There is already a shorthand

Reply:
I use to do this cuz don't know what "the next arg parser" is doing. :D